pkg/minishift/addon/command/echo_command.go

```diff
@@ -34,11 +34,19 @@ func NewEchoCommand(command string) *EchoCommand {
 }
 
 func (c *EchoCommand) doExecute(ec *ExecutionContext) error {
-	// split off the actual 'echo' command. As we need to print rest of the string
-	echo := strings.Replace(c.rawCommand, "echo ", "", 1)
-	echo = ec.Interpolate(echo)
+	var err error
 
-	_, err := fmt.Print("\n  " + echo)
+	// split off the actual 'echo' command. As we need to print rest of the string
+	echoString := strings.Replace(c.rawCommand, "echo", "", 1)
+
+	//Remove leading and trailing spaces
+	echoString = strings.TrimSpace(echoString)
+	if echoString == "" {
+		_, err = fmt.Print("\n")
```

Comment:
@praveenkumar I think it is almost same thing. I used `Print("\n")` for consistency in the existing code and it is explicit in a way.